index.html

```diff
@@ -0,0 +1,285 @@
+        <!-- nav -->
+        <header>
+        <div class="nav container">
+            <!-- Logo -->
+          <nav class="navbar">
+            <a href="#">Maher</a>
+          </nav>
+          <div class="end">
+            <img class="endnavbar" src="./img/Union.png" alt="Menu">
+          </div>
+          </div>
+        </header>
+     <!-- Home -->
+     <section>
+        <div class="home-content">
+          <h1>Maher Ramadan</h1>
+        </div>
+            <div class="home-text">
+              <h2>Software Developer</h2>
+            </div>  
+            <div class="paragraph">
+              <p>I can help you build a product,
+                feature or website Look through 
+                some of my work and experience! If  you like what you see 
+                and have a project that needs coding,don’t
+                 hesitate to  contact me. </p>
+            </div>
+            <div class="hr" style="padding:20px; margin: 20px; color: dark">
+              <hr>
+            </div>
+       </section>
+    <!-- line1 -->
+    <div class="list-technologies">
+    <div class="languages"> 
+      <h4>Languages</h4>
+      <ul class="menu-language">
+        <li>JavaScript</li>
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>CSS</li>
+      </ul>
+    </div>
+
+    <div class="Frameworks"> 
+      <h4>Frameworks</h4>
+      <ul class="menu-frameworks">
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>RSPec</li>
+        <li>CapyBara</li>
+        <li>Selenium</li>
+      </ul>
+    </div>
+
+    <div class="Skills">
+      <h4>Skills</h4>
+      <ul class="menu-skills">
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </div>
+  </div>
+
+      
+    
+
+<!-- line2 -->
+  <div class="hr" style="padding:20px; margin: 20px; color: dark">
+    <hr>
+  </div>
+            <!-- social media -->
+            <div class="section-one-column2">
+              <ul class="socialmedia">
+                  <li><a href=""><img src="./img/Vector.png" alt="github"></a></li>
+                  <li><a href=""><img src="./img/Frame.png"alt="Linkedin"></a></li>
+                  <li><a href=""><img src="./img/Vector5.png" alt="angel"></a></li>
+                  <li><a href=""><img src="./img/twitter.png" alt="twitter"></a></li>
+                  <li><a href=""><img src="./imG/M.png"alt="medium"></a></li>
+              </ul>
+            </div>
+
+            <!-- flesh down -->
+            <div class="flesh-down">
+              <img src="./img/Group 55.png" alt="flesh-down">
+              <img src="./img/Work.png" alt="work">
+            </div>
+
+            <!-- My recent work -->
+            <div class="My-Recent-Work">
+              <h3>MY RECENT WORK</h3>
+            </div>
+          
+            <!-- line3 -->
+            <div class="hr" style="padding:20px; margin: 20px; color: dark">
```

Comment:
- [ ] Kindly make sure that you place all your styles in `.css` files and avoid using inline styles